.pax/post-packaging.sh

```diff
@@ -268,6 +278,13 @@ mv "${SMPE_BUILD_SHIP_DIR}/${SMPE_PROMOTE_TAR}" ${SMPE_PROMOTE_TAR}
 echo "mv fmid.zip ${SMPE_FMID_ZIP}" > rename-back.sh.1047
 iconv -f IBM-1047 -t ISO8859-1 rename-back.sh.1047 > rename-back.sh
 
+#PSI creation from SMPE
+run_PSI_create=true
```

Comment:
You need to know where the SMPE build placed the rel-fales & SMPMCS to use as input for the PSI creation. Like the SMPE build, you also need to have a config file to manage where the PSI work area is, and where the output goes. 
Instead of driving a new script from post-packaging.sh, PSI creation could be triggered from smpe.sh with a startup argument (test "$run_PSI_create" = true && opt="$opt -S"). Doing the invocation here allows you to reuse get-config.sh to get the SMPE variables (and append it to add your own).

The way I envision it, if smpe.sh receives the -S argument, it executes 2 additional scripts, smpe-install-smpe.sh to install the smpe image, and smpe-psi.sh to create the PSI from this image. Finally post-packaging.sh can pick up the psi file and stage it for shipment to artifactory.

The PSI build could use the SMPE config setup for this, and instead of driving a new script from post-packaging.sh, PSI creation could be triggered from smpe.sh with a

Reply:
I have tried that based on this comment in new branch. Just was not sure about variable run_PSI_create (this was dummy one from initial commit, not part of pullReq), so I have used $BUILD_PSI" == "yes" which I have added to Jenkins trigger in later commit. Also I was not sure about smpe-install-smpe.sh, did you mean by that smpe-install.sh script, or some new one which does not exist now?

Reply:
> You need to know where the SMPE build placed the REL-files & SMPMCS to use as input for the PSI creation. Like the SMPE build, you also need to have a config file to manage where the PSI work area is, and where the output goes.
> Instead of driving a new script from post-packaging.sh, PSI creation could be triggered from smpe.sh with a startup argument (test "$run_PSI_create" = true && opt="$opt -S"). Doing the invocation here allows you to reuse get-config.sh to get the SMPE variables (and append it to add your own).
> 
> The way I envision it, if smpe.sh receives the -S argument, it executes 2 additional scripts, smpe-install-smpe.sh to install the smpe image, and smpe-psi.sh to create the PSI from this image. Finally post-packaging.sh can pick up the psi file and stage it for shipment to artifactory.

There, https://github.com/zowe/zowe-install-packaging/pull/2296#issuecomment-901222902